.drone.yml

```diff
@@ -135,6 +136,7 @@ steps:
 - name: build-client-contracts
   image: casperlabs/node-build-u1804
   commands:
+    - make audit-as
```

Comment:
Likewise; shouldn't be necessary as it is run in pre-checks

Reply:
I will change back the pipeline as per the discussion above.